minard/templates/burst.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+{% block title %}Burst Trigger{% endblock %}
+{% block head %}
+  {{ super() }}
+  <style>
+    .horizon .title
+  </style>
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12" width: 70%>
```

Comment:
So if you want this to be shorter than the whole page I think the way to do this is via:

```
<div class="col-md-8 col-md-offset-2">
```

which will make a column 8 units wide (about 70% of 12) and offset it by 2 (which will center it).